index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio setup mobile</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="toolbar">
+    <a href="#" class="logo">My logo</a>
+    <nav class="navbar">
+      <div class="hambergur-button">
+        <img src="images/Icon - Menu.png" alt="hambergur-icon">
+      </div>
+    </nav>
+  </header>
+  <main class="main-content">
+    <div class="text-content">
+      <h1 class="heading-1">
+        I'm Mariane
+        <br>
+        Glad to see
+        <br>
+        you!
+      </h1>
+
+      <h2 class="heading-2">I'am Shaheryar</h2>
+
+      <p class="paragraph">
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social-media">
+        <p class="paragraph-2">
+          LET'CONNECT!
+        </p>
+
```

Comment:
- [ ] As required for this project every subtitle should be `h2`or `h2`,.. but not a paragraph, kindly change this for semantic reasons. :wink: